platformio.ini

```diff
@@ -378,6 +385,96 @@ extra_scripts =
     ${board:ttgo-t-display.extra_scripts}
     ${common:prog_ota.extra_scripts}
 
+
+; ********************************************************************************
+; TTGO T-Display S3 ESP32 WiFi and Bluetooth Module Development Board - Programming via USB
+; ********************************************************************************
+[env:ttgo-t-display-S3-usb]
+extends = board:ttgo-t-display-S3, common:prog_usb, mode:selected
+platform = espressif32
+build_flags =
+    ${board:ttgo-t-display-S3.build_flags}
+    ${mode:selected.build_flags}
+extra_scripts =
+    ${board:ttgo-t-display-S3.extra_scripts}
+    ${common:prog_usb.extra_scripts}
+
+; ********************************************************************************
+; TTGO T-Display ESP32 WiFi and Bluetooth Module Development Board
+; ********************************************************************************
+[board:ttgo-t-display-S3]
+extends = common:esp32_env, display:ttgo_tdisplay-S3
+board = esp32-s3-devkitc-1
+board_build.partitions = default_8MB.csv
+board_build.filesystem = littlefs
+board_upload.flash_size = "8MB"
+board_upload.maximum_size = 8388608
+build_flags =
+    ${common:esp32_env.build_flags}
+    ${display:ttgo_tdisplay-S3.build_flags}
+    -DBOARD_LILYGO_T_DISPLAY_S3
+lib_deps =
+    ${common:esp32_env.lib_deps_builtin}
+    ${common:esp32_env.lib_deps_external}
+    ${display:ttgo_tdisplay-S3.lib_deps_builtin}
+    ${display:ttgo_tdisplay-S3.lib_deps_external}
+lib_ignore =
+    ${common:esp32_env.lib_ignore_builtin}
+    ${common:esp32_env.lib_ignore_external}
+    ${display:ttgo_tdisplay-S3.lib_ignore_builtin}
+    ${display:ttgo_tdisplay-S3.lib_ignore_external}
+extra_scripts =
+    pre:./scripts/get_git_rev.py
+
+
+
+; ********************************************************************************
+; TTGO T-Display S3
+; https://github.com/Xinyuan-LilyGO/T-Display-S3
+; ********************************************************************************
+[display:ttgo_tdisplay-S3]
```

Comment:
Please move the display part under the other display configurations.

Reply:
fixed as suggested.